src/megaclient.cpp

```diff
@@ -39,7 +39,8 @@ namespace mega {
 bool MegaClient::disablepkp = false;
 
 // root URL for API access
-string MegaClient::APIURL = "https://g.api.mega.co.nz/";
+// TODO: restore this before merging - needed for now for jenkins runs
+string MegaClient::APIURL = "https://lu1.api.mega.co.nz:444/";
```

Comment:
REMINDER :mega: :mega: : do not resolve this discussion until this changes has been restored